<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWindow</class>
 <widget class="QDialog" name="DeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="device_choose"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="device_info">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
